include/rocksdb/db.h

```diff
@@ -1447,6 +1448,12 @@ class DB {
   // DisableManualCompaction() has been called.
   virtual void EnableManualCompaction() = 0;
 
+  // Wait for Compaction to finish
```

Comment:
Based on what is said here, it's not clear what this does if there's no compaction in progress or pending. (Same feedback I had on #10524 from another author.) Does it also wait for flushes? Does it wait for all current pending-but-not-running compactions to finish? What about compactions triggered by compactions that are pending or running?

Assuming it waits for an instant in which there are no pending or running flushes or compactions, I think it's worth pointing out that this function might never return if there are sufficient ongoing writes. And if it does return, the state of "no compactions" could be in the past and have changed by the time the function returns to the caller. Though if the user has ceased all writes to the DB, it should eventually return in a stable state (with some minor exceptions like periodic compaction).